lol-qwop.qu

```diff
@@ -0,0 +1,2 @@
+'lol-qwop'
+test
```

Comment:
Based on https://en.wikipedia.org/wiki/Qu_(programming_language) shouldn't this be something like
```qu
println("lol-qwop")
```